<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogIn</class>
 <widget class="QDialog" name="LogIn">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1440</width>
    <height>780</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 249, 240);

</string>
  </property>
  <widget class="QFrame" name="LI_Header">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1440</width>
     <height>105</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: qlineargradient(
    spread:pad, 
    x1:0, y1:0, x2:1, y2:0, 
    stop:0 #EDC27E, 
    stop:0.526 #EDC27E, 
    stop:1 #DC586D
);

</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="LI_HeaderIcon">
    <property name="geometry">
     <rect>
      <x>-70</x>
      <y>-20</y>
      <width>274</width>
      <height>146</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Handwriting</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(98, 65, 66);
background: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../Downloads/HeaderIcon.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="LI_MainPanel">
   <property name="geometry">
    <rect>
     <x>182</x>
     <y>136</y>
     <width>1082</width>
     <height>602</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 240, 216);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QFrame" name="LI_SidePanel">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>0</y>
      <width>421</width>
      <height>602</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 214, 182);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="LI_SidePanelBg">
     <property name="geometry">
      <rect>
       <x>-310</x>
       <y>0</y>
       <width>1071</width>
       <height>601</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../Downloads/LI_SidePanel.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="LI_StatementLBL">
     <property name="geometry">
      <rect>
       <x>-140</x>
       <y>80</y>
       <width>721</width>
       <height>421</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(122, 12, 12);
background: transparent;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../Downloads/LI_SignUpStatement.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="LI_SignUpPB">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>370</y>
       <width>119</width>
       <height>36</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <underline>true</underline>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border-color: rgb(122, 12, 12);
color: rgb(122, 12, 12);
border: 3px solid #550808;
background-color: rgb(229, 141, 118);
border-radius: 10px;
text-decoration: underline;
box-shadow: 5px 5px 15px rgba(0, 0, 0, 0.5);
text-color: #E58D76;

</string>
     </property>
     <property name="text">
      <string>Sign Up</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="LI_LogInToPenpal">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>150</y>
      <width>411</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>36</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(122, 12, 12);
background: transparent;</string>
    </property>
    <property name="text">
     <string>Log In to Penpal</string>
    </property>
   </widget>
   <widget class="QLabel" name="LI_UsernameLBL">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>171</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(122, 12, 12);
background: transparent;</string>
    </property>
    <property name="text">
     <string>Username:</string>
    </property>
   </widget>
   <widget class="QLabel" name="LI_PasswordLBL">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>320</y>
      <width>171</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(122, 12, 12);
background: transparent;</string>
    </property>
    <property name="text">
     <string>Password:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LI_PasswordLE">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>380</y>
      <width>441</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(229, 141, 118);
border: 3px solid #E58D76;
border-radius: 5px;
background: transparent;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LI_LogInPB">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>480</y>
      <width>119</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(229, 141, 118);
border: 3px solid #E58D76;
border-radius: 10px;
text-decoration: underline;
text-color: #E58D76;
box-shadow: 5px 5px 15px rgba(0, 0, 0, 0.5);
background: transparent;
</string>
    </property>
    <property name="text">
     <string>Log In</string>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="LI_Icon">
    <property name="geometry">
     <rect>
      <x>-40</x>
      <y>-10</y>
      <width>274</width>
      <height>146</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Handwriting</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(98, 65, 66);
background: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../Downloads/Icon.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="LI_UsernameLE">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>280</y>
      <width>441</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(229, 141, 118);
border: 3px solid #E58D76;
border-radius: 5px;
background: transparent;</string>
    </property>
   </widget>
   <widget class="QLabel" name="LI_ForgotPasswordLBL">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>430</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
      <underline>true</underline>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(122, 12, 12);
background: transparent;
text-decoration: underline
</string>
    </property>
    <property name="text">
     <string>Forgot Password?</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>0</y>
      <width>941</width>
      <height>671</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../Downloads/LI_Shadow.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>LI_SidePanel</zorder>
   <zorder>LI_Icon</zorder>
   <zorder>LI_ForgotPasswordLBL</zorder>
   <zorder>LI_PasswordLE</zorder>
   <zorder>LI_PasswordLBL</zorder>
   <zorder>LI_UsernameLE</zorder>
   <zorder>LI_UsernameLBL</zorder>
   <zorder>LI_LogInToPenpal</zorder>
   <zorder>LI_LogInPB</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
